prep-javascript-for-loops/main.js

```diff
@@ -0,0 +1,15 @@
+var i = '';
+
+for (i = 0; i < 10; i++) {
+  console.log(i);
+}
+
+var i = '';
+
+for (i = 0; i < 10; i++) {
+  console.log(i * 2);
+}
+
+for (i = 100; i > 0; i--) {
```

Comment:
don't forget to declare the variables `var i = 100`

Reply:
![updated-js-for-loops](https://user-images.githubusercontent.com/118767169/213555880-2ba97ea6-4054-4bd8-b288-46e9155b740c.png)